src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,34 @@
+package pages;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+public class HomePage extends BasePage {
+
+    By userAvatarIcon = By.cssSelector("img.avatar");
+    By firsPlaylist = By.cssSelector(".playlist:nth-child(3)");
+    By playlistInputField = By.cssSelector("input[name='name']");
```

Comment:
I recommend:  make all the locators **private**, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

If a variable is used only by other methods of that class, it should be declared as private.

For example

    ```
    private By userAvatarIcon = By.cssSelector("img.avatar");
    private By firsPlaylist = By.cssSelector(".playlist:nth-child(3)");
    private By playlistInputField = By.cssSelector("input[name='name']");

Reply:
I didn't know, thanks for the advice))